DXMainClient/Domain/Multiplayer/CnCNet/HostedCnCNetGame.cs

```diff
@@ -33,9 +33,9 @@ public class HostedCnCNetGame : GenericHostedGame
 
         public override int Ping => TunnelServer.PingInMs;
 
-        public override bool Equals(GenericHostedGame other) => 
-            other is HostedCnCNetGame hostedCnCNetGame ? 
-                string.Equals(hostedCnCNetGame.ChannelName, ChannelName, StringComparison.InvariantCultureIgnoreCase) :
-                base.Equals(other);
+        public override bool Equals(GenericHostedGame other)
+            => other is HostedCnCNetGame hostedCnCNetGame
+                ? string.Equals(hostedCnCNetGame.ChannelName, ChannelName, StringComparison.InvariantCultureIgnoreCase)
+                : base.Equals(other);
```

Comment:
I have a (hopefully pretty convincing) reasoning for this style/change:
1) having expression body arrow on the line after the function prototype makes the function prototype not changed in case of change to/from expression body to traditional method definition, reducing merge conflicts in the long run;
2) because we read English and source code from left to right, having operators on the beginning of the line allows to see what's written way more easy and fast than having them at the end of the line, especially when the line is long as we can see in this case (plus, the beginnings of the lines are aligned, so it's even easier to see what's going on compared to staggered endings of the lines).